source/acquaman/SGM/SGMXASScanConfiguration.cpp

```diff
@@ -94,9 +94,9 @@ AMScanConfigurationView* SGMXASScanConfiguration::createView(){
 }
 
 QString SGMXASScanConfiguration::detailedDescription() const{
-	double exitSlit;
-	double grating;
-	double harmonic;
+	double exitSlit = 0;
+	double grating = 0;
+	double harmonic = 0;
```

Comment:
Looking at this now I'm wondering if there is actually a chance that there might not be a match found for each of these, and if this is the case whether we should catch that in the formatting of the string at the bottom. 
After all if there is no element of the group found with the exit slit gap name an output that indicates the exit slit gap was set to 0 would be included, which isn't technically correct. 

Of course if the group is guaranteed to contain these elements it is a non issue. 
@davidChevrier Is this guarantee the case?